.gitignore

```diff
@@ -34,3 +34,6 @@ certs
 
 # Typescript build info
 *.tsbuildinfo
+
+# Bruno: Abdul testing out bruno
+bruno/
```

Comment:
[We don't talk about Bruno 🎵](https://www.youtube.com/watch?v=bvWRMAU6V-c)

Reply:
LOL 🤣  will remove